index.html

```diff
@@ -12,14 +12,28 @@
 			<img src="images/logo.png" alt="the network"/>
 		</div>	
 		<div id="search">
-			<form method="get">
-				<input type="text" name="post" placeholder="thoughts?"/>
+			<form onsubmit="submitFunction(event)" method="get">
```

Comment:
This looks like it will work but, the second requirement of the test stipulates not changing the form markup.
"2.Without changing the existing form markup in index.html, create a script that on form submit prepends the contents of the from"

Reply:
@LorenzoGangi I misunderstood that part. I removed the form changes.

Reply:
Thanks contact recruiting@ontraport.com to do an HR screening.